src/ansys/edb/core/layout/cell.py

```diff
@@ -295,7 +295,7 @@ def design_mode(self):
 
     @design_mode.setter
     def design_mode(self, value):
-        self.__stub.SetDesignMode(messages.design_mode_property_message(self, value))
+        self.__stub.SetDesignMode(messages.design_mode_property_message(self.msg, value))
```

Comment:
nitpick: seems better to do `target.msg` inside `design_mode_property_message` method (at least from the naming) but very minor